### ISSUE TITLE
HTTPS

### ISSUE THREAD

**Issue by NielsLiisberg** [sitemule/ILEastic#26]
SSL / TLS would be nice if services are run with a public interface.

**Comment by jwoehr** [comment]
I agree completely. On IBMiOSS on Ryver, Richard Schoen suggested perhaps this should be done by an Apache reverse proxy.

**Comment by NielsLiisberg** [comment]
Absolutely - However SSL/TLS for HTTPS is a plan for the future ...The Apache / Nginx revese proxy is the solution for now

**Comment by wimjongman** [comment]
One of the nice features of ILEastic is its capability to ditch the middle man... Having to use an RP negates some of that.

**Comment by NielsLiisberg** [comment]
.. Actually - the SSL/TLS code is in the project now. I have, however, not made it active yet...

**Comment by krka01** [comment]
Sorry to bump this old thread, but ss the SSL/TLS code active in the project yet?
I guess not as this issue is still in open status.

**Comment by NielsLiisberg** [comment]
No - it is not active yet - We use ILEastic primary as exposing RPGLE in a
microservice environment, where we normally use NGINX to dispatch requests
over plain HTTP.

However we can at system & method A/S enable it for you, if you have a
urgent need.

**Comment by krka01** [comment]
Thanks for the info. No, it is not urgent as we are in an evaluating phase for different web service solution.

Using NGINX as a reverse proxy could be a solution.

 Is activating TLS in ileastic sometime in the future still on the agenda and if so, is there any ETA?

**Comment by wimjongman** [comment]
I hope you are considering Icebreak from the same provider that gave you ileastic, https://system-method.com/icebreak

**Comment by NielsLiisberg** [comment]
Thanx Wim for pointing Krister in that direction. I try to be "neutral"
here in the open source space. But certainly IceBreak will do what Krister
is asking for. Both ILEastic, noxDb and ILEvator are initially carved out
of the enterprise version of IceBreak with courtesy of System & Method -
Thanks to them. So IceBreak is a valid answer to the question.

**Comment by m1h43l** [comment]
Added with PR #169 .
